### DISCUSSION TITLE
`contenteditable="plaintext-only"`

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#155] [State of HTML Features]
Permits editing of the element's raw text, but not rich text formatting.

### Code sample

```html
<h2 class="title" contenteditable="plaintext-only"></h2>
```

### MDN URL:

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable

### Specification URL:

https://html.spec.whatwg.org/multipage/interaction.html#attr-contenteditable

### How stable is this feature?

Cutting edge (Shipped, but not in all browsers)

### Type of feature

HTML attribute value

### What focus area does this feature fall under?

Forms

**Comment by Que-tin** [comment]
I'm not sure if we should ask for the `plaintext-only` value in particular rather than asking for the familiarity with `contenteditable` in general. I feel like even though it is not that widely known.
